skimage/measure/_label.py

```diff
@@ -38,6 +38,10 @@ def label(input, neighbors=None, background=None, return_num=False,
         as a neighbor.
         Accepted values are ranging from  1 to input.ndim. If ``None``, a full
         connectivity of ``input.ndim`` is used.
+    ratio : int, optional
```

Comment:
I have to admit that I do not understand what's performed after reading the docstring...